### PR TITLE
fixed typo in name class MessagePopup

### PR DESCRIPTION
**Pull request by atzurk**
corrected class name to MessagePopup (getting rid of the triple s)

### CODE REVIEW

**Review by olgapuig — CHANGES_REQUESTED** [review]
This also needs to be changed in the Dialog as we reviewed together.

**Comment by atzurk** [comment]
> This also needs to be changed in the Dialog as we reviewed together.

doing it, thanks!

**Review by olgapuig — APPROVED** [review]
Thanks for the contribution Marco.

**Review by olgapuig — CHANGES_REQUESTED** [review]
Increase the version in the package.json to be able to publish the fix

**Comment by atzurk** [comment]
> Increase the version in the package.json to be able to publish the fix

ok, updated and pushed. Ready to publish!

**Review by olgapuig — APPROVED** [review]

File: src/widgets/dialog.ts [hunk 1]
```diff
@@ -31,7 +31,7 @@ export class Dialog extends Widget {
 			.element(by.buttonText(name)));
 	}
 
-	public getMesssagePopup(): MesssagePopup {
-		return new MesssagePopup();
+	public getMesssagePopup(): MessagePopup {
```
Comment (Franek75) [inline src/widgets/dialog.ts:34]: You could drop the 3rd s here, also :)

**Review by fperezluna — APPROVED** [review]

File: src/widgets/dialog.ts (hunk 1 above)
Comment (atzurk) [reply src/widgets/dialog.ts]: oops, sure I can... thanks for your review!!

**Pull request merged**